README.md

```diff
@@ -12,4 +12,3 @@ In this milestone I added the 3 main component (Display , ButtonPanel and Button
 # Live Link:
 [Live Link](https://dry-crag-03357.herokuapp.com/)
```

Comment:
- Update the readme file with instructions for getting started, installing packages, etc. Check out this template for reference: [template](https://github.com/microverseinc/readme-template)